css/main.css

```diff
@@ -168,40 +161,82 @@ kbd {
 }
 
 .contentContainer{
+	/*inner stuff*/
 	display: flex;
-	min-width: 0;
-	min-height: 0;
+	margin: 0 0.5em;
```

Comment:
I like having the contentContainer itself be 0 margin on all sides so that the rightmost scroll bar has no gap (this also fixes an uneven margin issue present on the master by doing this). The only downside is text outside of the collapsible will be touching the edge of the window but maybe having a div class that wraps all of that text and pads it could solve it.